### ISSUE TITLE
Parameter to limit the number of faces in the scene 

### ISSUE THREAD

**Issue by OndraZizka** [ORB-HD/deface#7]
Sometimes there's a scene with a fixed number of people.  
So there is an assumption they each have 1 face.  

By having such parameter, the false positives could be reduced.

**Comment by mdraw** [comment]
Good idea :+1: A maximum setting would be easy to implement, something like:
`--max-faces=k`: Face detection candidates are sorted by their scores and only the top k candidates are selected. Does nothing if there are less than k candidates.

Instead of implementing a setting for the or exact number of expected faces (if such a need arises), I would rather recommend setting the `--thresh` to a very low number (0.1 or 0.05 - smaller numbers slow down inference too much) and setting `--max-faces` to the expected number of hits.

Would you like to implement this? Otherwise I'll work on it myself in the near future.

**Comment by OndraZizka** [comment]
I could have a look at implementing it, but first around autumn this year. So, if "near future" is before that, you'll be quicker.
